servlet/6.0/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Jakarta Servlet 6.0 (under development)"
+date: 2021-10-15
```

Comment:
This date controls when the page is rendered, once this PR is pushed. Please set it to the merge date, or earlier. Otherwise, this page will not appear until Oct 15 which is not what we want.